appium/webdriver/webdriver.py

```diff
@@ -523,9 +523,25 @@ def update_settings(self, settings):
 
     @property
     def device_time(self):
-        """Returns the date and time from the device
+        """Returns the date and time from the device.
         """
-        return self.execute(Command.GET_DEVICE_TIME, {})['value']
+        return self.execute(Command.GET_DEVICE_TIME_GET, {})['value']
+
+    def get_device_time(self, format=None):
+        """Returns the date and time from the device. (Only available since Appium 1.11.0)
+
+        :Args:
+         - format - The default format is `YYYY-MM-DDTHH:mm:ssZ`, which complies to ISO-8601
+           The set of format specifiers. Read https://momentjs.com/docs/ to get
+           the full list of supported datetime format specifiers.
+
+        :Usage:
+            self.driver.get_device_time("YYYY-MM-DD")
+        """
+        if format:
+            return self.execute(Command.GET_DEVICE_TIME_POST, {'format': format})['value']
+        else:
```

Comment:
if format is None (https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html)

also, `else` is redundant

Reply:
```
if format is None:
    return self.device_time
return self.execute(Command.GET_DEVICE_TIME_POST, {'format': format})['value']
```
is better than below?
```
if format:
    return  self.execute(Command.GET_DEVICE_TIME_POST, {'format': format})['value']
return self.device_time
```

>also, else is redundant

oh,,,thanks.

Reply:
Done by https://github.com/appium/python-client/pull/312/commits/54815e8dd04947a2738ddc98ecb3e7cecdebec02

Reply:
The PR indicates the default argument is None. It is good to handle return value following the None then. None is better than `""`, since we can compare them as literal explicitly.